rendering_resource_manager_service/session/management/job_manager.py

```diff
@@ -111,10 +112,12 @@ def schedule(self, session, params, environment):
                                              parameters, environment_variables, modules)
             session.status = SESSION_STATUS_SCHEDULED
             session.save()
-            return [200, 'Job ' + str(session.job_id) + ' now scheduled']
+            response = json.dumps({'contents': 'Job ' + str(session.job_id) + ' now scheduled'})
```

Comment:
it would be nicer to have job_id with its own attribute:
{
'message': 'Job scheduled',
'jobId': session.job_id
}

same thing for the others job_id occurence